tests/returntypes/transforms/test_metric_tensor_new.py

```diff
@@ -123,12 +124,11 @@ def test_construct_subcircuit(self):
         assert isinstance(tapes[1].operations[3], qml.Hadamard)
 
         # third parameter subcircuit
-        assert len(tapes[2].operations) == 4
+        assert len(tapes[2].operations) == 3

```

Comment:
What happened here?

Reply:
Because of the `return` statement in the tape construction, this test was not executed at all. It simply ended at the line 
```python
return qml.expval(qml.PauliX(0)), qml.expval(qml.PauliX(1))
```
because there is no internal function to end, but the function `test_construct_subcircuit` :grimacing: